### PR TITLE
Format Dates

### PR DESCRIPTION
**Pull request by steven-solar**
This PR formats all the dates consistently using moment.js. 

Screenshot 2020-08-01 at 6 22 48 PM

### CODE REVIEW

File: capstone-frontend/src/components/AssignmentCard.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { Button, Col, Form, Row } from 'react-bootstrap';
-
+import moment from 'moment';
```
Comment (MauroGuerrero20) [inline capstone-frontend/src/components/AssignmentCard.jsx:3]: What is the moment import?

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

**Review by antoniolinhart — APPROVED** [review]
Looks good! This makes the dates much cleaner

File: capstone-frontend/src/components/AssignmentCard.jsx (hunk 1 above)
Comment (steven-solar) [reply capstone-frontend/src/components/AssignmentCard.jsx:3]: This is the npm package that formats dates.

**Review by VivekShankar — APPROVED** [review]
LGTM - the new date format looks good!

**Review by geoffbass — APPROVED** [review]
LGTM, if you're using a consistent format across components (or even just several times within a component) you could use a util function or format string constant

**Pull request merged**